transport/grpc/dial.go

```diff
@@ -276,8 +276,8 @@ func isDirectPathEnabled(endpoint string) bool {
 	}
 
 	// Only try direct path if the user has opted in via the environment variable.
-	whitelist := strings.Split(os.Getenv("GOOGLE_CLOUD_ENABLE_DIRECT_PATH"), ",")
-	for _, api := range whitelist {
+	allowlist := strings.Split(os.Getenv("GOOGLE_CLOUD_ENABLE_DIRECT_PATH"), ",")
```

Comment:
Perhaps `directPathAPIs`? More clear as to what it is.

Reply:
Ack.